static/js/components/dashboard/FinalExamCard.js

```diff
@@ -151,11 +151,27 @@ const renderPearsonTOSDialog = (open, show, submitPearsonSSO, pearson) => (
     autoScrollBodyContent={true}>
     <div className="dialog-container">
       <img src="/static/images/pearson_vue.png" width="180"/>
-      <h3 className="dialog-title">Test Registration is completed on the <br/>Pearson VUE website</h3>
-      <p>
-        I acknowledge that by clicking Continue I will be leaving the MicroMasters <br/>website and going to
-        the Pearson VUE website, and that I accept the <br/>Pearson VUE Group’s <a target="_blank"
-          href="https://home.pearsonvue.com/Legal/Privacy-and-cookies-policy.aspx">Terms of Service</a>.
+      <h3 className="dialog-title">You are being redirected to Pearson VUE’s website.</h3>
+      <p className="tos-container">
+        You acknowledge that by clicking Continue, you will be leaving the MITx MicroMasters website
+        and going to a third-party website over which MIT’s MITx does not have control, and that you
+        accept the Pearson VUE Business Group’s Terms of Service. MIT is not responsible for the content
```

Comment:
I think we should like to the Pearson TOS here (as we do currently) so that users can go read that before they affirm they agree to it. @roberthouse54 what do you think?